addons/custom-zoom/style.css

```
@@ -4,11 +4,11 @@
   right: 0;
   width: 60px;
   height: 148px;
-  pointer-events: none;
+  z-index: 21;
```

Can you explain the choice of 21?

---

Not sure why I set it to that, it works perfectly fine if it's lower than that.

```suggestion
  z-index: 1;
```